docs/index.yml

```diff
@@ -0,0 +1,193 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: YnvestTubeApi
+host: "127.0.0.1:8000"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /users:
+    get:
+      summary: 'Get all users'
+      tags:
+        - user
+      responses:
+        '200':
+          description: 'Users list retrieved successfully'
+          schema:
+            $ref: '#/definitions/UserList'
+        '403':
+          description: Bad user id
+  /users/register:
+    get:
+      summary: "Register new user"
+      tags:
+        - user
+      responses:
+        "200":
+          description: 'User id for an app to use in future calls'
+          schema:
+            $ref: '#/definitions/UserId'
+  /users/{userId}:
+    get:
+      summary: "Get user's overall info cash, account creation date."
+      tags:
+        - user
+      parameters:
+        - in: path
```

Comment:
I am not sure if placing UUID in request path is a good pattern. But it is just a small reflection. I doesn't need to be changed. If it is stays like that it surely would't be a diseaster.